biz.aQute.junit/src/aQute/junit/ConsoleReporter.java

```diff
@@ -0,0 +1,53 @@
+package aQute.junit;
+
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+import aQute.lib.exceptions.Exceptions;
+import junit.framework.AssertionFailedError;
+import junit.framework.Test;
+import junit.framework.TestListener;
+
+public class ConsoleReporter implements TestListener {
```

Comment:
Maybe I should get rid of the Tee streams because they are conflating different concerns. I feel multiple listeners is the right model for handling different use cases of the test output.